pkgs/development/libraries/ffmpeg/options.nix

```diff
@@ -0,0 +1,752 @@
+{
+  lib,
+  config,
+  stdenv,
+  pkgs, # TODO
+  version,
+  variant,
+  variants,
+  ...
+}:
+
+
+let
+  # Checks whether the given variant of the flag is contained in the currently
+  # active variant. full ⊃ small ⊃ headless
+  isInVariant = flagVariant:
+    let
+      checks = {
+        full = variant == variants.full;
+        small = variant == variants.small || checks.full;
+        headless = variant == variants.headless || checks.small;
```

Comment:
Shouldn't the checks be:
- headless: check only headless
- small: check both headless & small
- full: check all
?

Reply:
The function works the other way around: checking whether the feature should be enabled in a certain variant. Any feature that is enabled in headless should also be enabled in full but not vice versa.

This is analogous to how `with*Deps` worked before.